3-networks-dual-svpc/envs/development/main.tf

```diff
@@ -73,6 +73,8 @@ module "base_env" {
   default_region2                    = local.default_region2
   domain                             = var.domain
   parent_folder                      = var.parent_folder
+  ingress_policies                   = var.ingress_policies
+  egress_policies                    = var.egress_policies
```

Comment:
Let's directly set them here as opposed to using a var.

Reply:
Hi @bharathkkb  do you mean?
```
  ingress_policies                   = []
  egress_policies                    = []
```

Reply:
Hi Barath, variables were added there to support some assertions on integrations tests. Should I still delete them?

Reply:
Sounds good, lets keep for now. Looks like also need a rebase.